public/data/kapost_v1.json

```diff
@@ -559,14 +559,14 @@
               "Required":"N",
               "Default":"",
               "Type":"utc date",
-              "Description":"Unix timestamp (in milliseconds, ex '1376092036000')<br/><br/><strong>Beta Warning</strong>: The format for this field will soon change to <a href='http://www.w3.org/TR/NOTE-datetime'>ISO 8601</a>."
+              "Description":"Unix timestamp (in milliseconds, ex '1376092036000')"
```

Comment:
This should say "in ISO 8601" (should probably keep the link) rather than "in milliseconds".